src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingPanels.md

```diff
@@ -110,17 +110,35 @@ fluid.defaults("fluid.prefs.panel.lineSpace", {
         wideIcon: ".flc-prefsEditor-line-space-wideIcon",
         multiplier: ".flc-prefsEditor-multiplier"
     },
+    selectors: {
+            lineSpace: ".flc-prefsEditor-line-space",
+            label: ".flc-prefsEditor-line-space-label",
+            narrowIcon: ".flc-prefsEditor-line-space-narrowIcon",
+            wideIcon: ".flc-prefsEditor-line-space-wideIcon",
+            multiplier: ".flc-prefsEditor-multiplier",
+            lineSpaceDescr: ".flc-prefsEditor-linespace-descr"
+    },
```

Comment:
There are two selectors blocks now. I think you meant to replace the old one. Also the tabbing is off in the new one.